src/main/java/frc/team2412/robot/test/AutomatedTestRobot.java

```diff
@@ -0,0 +1,70 @@
+package frc.team2412.robot.test;
+
+import edu.wpi.first.hal.simulation.DriverStationDataJNI;
+import edu.wpi.first.wpilibj.DriverStation;
+import frc.team2412.robot.Robot;
+
+public class AutomatedTestRobot extends Robot {
+	private static void sleep(long durationMillis) {
+		try {
+			Thread.sleep(durationMillis);
+		} catch (InterruptedException interrupt) {
+			System.out.println("Interrupted");
+		}
+	}
+
+	private static AutomatedTestRobot instance = null;
+
+	public static AutomatedTestRobot getInstance() {
+		if (instance == null) instance = new AutomatedTestRobot();
+		return instance;
+	}
+
+	private AutomatedTestRobot() {
+		System.out.println("Robot type: Automated test");
+	}
+
+	@Override
+	public void startCompetition() {
+		try {
+			super.startCompetition();
+		} catch (Throwable throwable) {
+			Throwable cause = throwable.getCause();
+			if (cause != null) {
+				throwable = cause;
+			}
+			DriverStation.reportError(
+					"Unhandled exception: " + throwable.toString(), throwable.getStackTrace());
+			// Sleep was added to original code in
+			// https://github.com/robototes/RapidReact2022/pull/43/commits/e9e81bcae76dabe08b6f2d24eb56b55378012565
+			// sleep(2000);
+			System.exit(-1);
```

Comment:
Please confirm you've verified that throwing an unhandled exception still causes a callstack to show up in the CI output. Pushing a commit with something set to `null`, running the CI tests, and then undoing that commit should be sufficient.

Reply:
I've tested behavior locally with `./gradlew simulateJava -PautomatedTest=true` (`simulateJavaDebug` doesn't work on school laptops), but it would be valuable to use the commit method.

Reply:
The log in CI is exactly the same as running it on your own system so I don't think it's really needed

Reply:
I'll still do them so that people could see the exact results, but yeah I don't expect it to be different from locally (unless `simulateJava` has some odd differences with `simulateJavaDebug`).